### PR TITLE
Standardize content format

### PR DESCRIPTION
**Pull request by jenmyers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +29,8 @@ In this lab, we're going to practice finding elements in the DOM. To do so, we'r
 </body>
 ```
 
-If we called `document.querySelector('div')`, the method would return the first `div` (whose `.innerHTML` is "Hello!").
+If we called `document.querySelector('div')`, the method would return the first
+`div` (whose `.innerHTML` is "Hello!").
```
Comment (sgharms) [inline README.md]: This feels like the .innerHTML is related to the selection's result. 

(whose _content_ is "Hello!"). 

?

File: README.md [hunk 2]
```diff
@@ -64,25 +69,40 @@ Selectors aren't limited to tag names, though (otherwise why not just use `docum
...
+US/docs/Web/Guide/CSS/Getting_Started/Selectors). Play around on the MDN page,
+then come back when you're ready.
 
 ### `querySelectorAll()`
 
-`querySelectorAll` works a lot like `querySelector()` -- it accepts a selector as its argument, and it searches starting from the element that it's called on (or from `document`) -- but instead of returning the _first_ match, it returns a NodeList (which, remember, is _not_ an Array) of matching elements.
+`querySelectorAll` works a lot like `querySelector()` -- it accepts a selector
+as its argument, and it searches starting from the element that it's called on
+(or from `document`) -- but instead of returning the _first_ match, it returns a
+NodeList (which, remember, is _not_ an Array) of matching elements.
```
Comment (sgharms) [inline README.md]: ...of *all* matching elements.

Use to contrast _first_ match in previous clause

File: README.md [hunk 3]
```diff
@@ -100,7 +120,9 @@ Given a document like
 </main>
 ```
 
-If we called `document.getElementById('app').querySelectorAll('ul.ranked-list li')`, we'd get back a NodeList of `<li>1</li>, <li>2</li>, <li>10</li>, <li>11</li>`.
+If we called `document.getElementById('app').querySelectorAll('ul.ranked-list
+li')`, we'd get back a NodeList of `<li>1</li>, <li>2</li>, <li>10</li>,
```
Comment (sgharms) [inline README.md]: Maybe we need to be more precise here. We're not getting back a NodeList of these strings, but rather the _Nodes_ corresponding to the elements.

File: README.md [hunk 4]
```diff
@@ -100,7 +120,9 @@ Given a document like
...
+<li>11</li>`.
 
 We could change the `.innerHTML` of these `li`s like so:
```
Comment (sgharms) [inline README.md]: I'm not sure where innerHTML is introduced, but is there a compelling reason to not swap all uses of it with _content_ versus innerHTML?

File: README.md [hunk 5]
```diff
@@ -112,13 +134,18 @@ for (let i = 0; i < lis.length; i++) {
 }
```
Comment (sgharms) [inline README.md]: Can we ensure that we've rolled out loops before this point?

**Comment by maxwellbenton** [comment]
Merging this in as is

**Pull request merged**